docs/user_guide/emummc/partitioning_sd.md

```diff
@@ -91,13 +105,12 @@
 !!! danger "SD card identification"
 	Make 100% sure you get your microSD card during the following steps. If you are not careful, you can end up wiping the wrong disk. We are not responsible for data loss when doing this.
 
-### What you will need (macOS)
-
-- Access to an administrator account.
+!!! tip "What you will need (macOS)"
```

Comment:
This does still need a second level heading.

Reply:
its a tip box, heading not needed.

Reply:
Was in coincidence with the linux instructions, hence the reasoning.